### PR TITLE
[RP-17] Handle Canada + US states/provinces

### PR DESCRIPTION
**Pull request by daaray**
- Adds support for Canada and US states/provinces
- Outputs summary data for countries represented

**Note:** This will require installing new requirements: `pip install -r requirements.txt`

### CODE REVIEW

File: dashboard.py [hunk 1]
```diff
@@ -17,23 +18,30 @@
 async def results(request):
     """Fetch remote results forward on."""
 
+    def _derive_mapping_code(response, mapped_totals):
+        """Consumes a list of dictionaries and returns a mapping of 2 char CC-SS (Country State)
+        keys and total submissions for each."""
+        values = response['values']
+        code = '{0}-{1}'.format([x['value'].lower() for x in values if x['label'] == 'country_code'][0], [x['value'].lower() for x in values if x['label'] == 'state_code'][0])  # noqa
```
Comment (daaray) [inline dashboard.py]: If someone has a better way to do this...let me know. This seems really ugly.

Comment (emullaney) [inline dashboard.py]: I don't have any other ideas and it is functional. It is really long wouldn't be bad to break it up to a few lines.

File: dashboard.py [hunk 2]
```diff
@@ -17,23 +18,33 @@
 async def results(request):
     """Fetch remote results forward on."""
 
+    def _derive_mapping_code(response, mapped_totals):
+        """Consumes a list of dictionaries and returns a mapping of 2 char CC-SS (Country State)
+        keys and total submissions for each."""
+        values = response['values']
+        code = '{0}-{1}'.format(
+            [x['value'].lower() for x in values if x['label'] == 'country_code'][0],
```
Comment (daaray) [inline dashboard.py]: For PollRuns that are incomplete, this may return an IndexError; need to account for this.

**Comment by emullaney** [comment]
Looks good. Will be interested to see how @kialam adds sparkle to the country bar graph.
:sparkler: :rainbow:

**Pull request merged**